content/docs/app-developer-guide/run-an-app.md

```diff
@@ -0,0 +1,139 @@
+The `launcher` will source any profile scripts (for `non-direct` processes) and set buildpack-provided environment variables in the app's execution environment before starting the app process.
+
+#### Default process type
+
+If you would like to run the default process, you can simply run the app image without any additional arguments:
+
+```bash
+docker run --rm -p 8080:8080 multi-process-app
+```
+
+#### Default process type with additional arguments
+
+If you would like to pass additional arguments to the default process, you can run the app image with the arguments specified in the command:
+
+```bash
+docker run --rm -p 8080:8080 multi-process-app --spring.profiles.active=mysql
+```
+
+#### Non-default process-type
+
+To run a non-default process type, set the process type as the entrypoint for the run container:
+
+```bash
+docker run --rm --entrypoint sys-info multi-process-app
+```
+
+#### Non-default process-type with additional arguments
+
+You can pass additional arguments to a non-default process type:
+
+```bash
+docker run --rm --entrypoint sys-info multi-process-app --spring.profiles.active=mysql
+```
+
+#### User-provided shell process
+
+You can even override the buildpack-defined process types:
+
+```bash
+docker run --rm --entrypoint launcher -it multi-process-app bash
+docker run --rm --entrypoint launcher -it multi-process-app echo hello "$WORLD" # $WORLD is evaluated on the host machine
+docker run --rm --entrypoint launcher -it multi-process-app echo hello '$WORLD' # $WORLD is evaluated in the container after profile scripts are sourced
+```
+
+#### User-provided shell process with bash script
+
+An entire script may be provided as a single argument:
+
+```bash
+docker run --rm --entrypoint launcher -it multi-process-app 'for opt in $JAVA_OPTS; do echo $opt; done'
+```
+
+#### User-provided direct process
+
+By passing `--` before the command, we instruct the `launcher` to start the provided process without `bash`.
+Note that while buildpack-provided environment variables will be set in the execution environment, no profile scripts will be sourced (as these require `bash`):
+
+```bash
+docker run --rm --entrypoint launcher multi-process-app -- env # output will include buildpack-provided env vars
+docker run --rm --entrypoint launcher multi-process-app -- echo hello "$WORLD" # $WORLD is evaluated on the host machine
+docker run --rm --entrypoint launcher multi-process-app -- echo hello '$WORLD' # $WORLD is not evaluated, output will include string literal `$WORLD`
+```
+
+#### No launcher
+
+You can bypass the launcher entirely by setting a new entrypoint for the run container:
+
+```bash
+docker run --rm --entrypoint bash -it multi-process-app # profile scripts have NOT been sourced and buildpack-provided env vars are NOT set in this shell
+```
+
+To learn more about the launcher, see the [platform spec](https://github.com/buildpacks/spec/blob/main/platform.md#launcher).
+
+[samples]: https://github.com/buildpacks/samples
```

Comment:
Did you mean to put a link to `samples`? If yes, I think parenthesis are missing.

Reply:
These are link references so that you can then use them throughout the doc as `[samples][samples]`.

Reply:
I didn't know. Thank you @jromero!